CHANGELOG.md

```diff
@@ -2,6 +2,9 @@
 
 ## Next Release
 
+### Interactivity improvements
+- Send message by pressing <kbd>ctrl</kbd>+<kbd>d</kbd> along with <kbd>meta</kbd>+<kbd>enter</kbd>.
```

Comment:
Perhaps simpler, like the following?
`Add additional shortcut key for sending messages (<kbd>ctrl</kbd>+<kbd>d</kbd>)`

We should probably pick a consistent term to use, ie. 'hot keys` vs `keyboard shortcuts` vs `shortcut keys`, etc